modules/rosa-sts-cluster-terraform-file-creation.adoc

```diff
@@ -233,7 +350,7 @@ module "operator_roles" {
 EOF
 ----
 
-.. Create the `variables.tf` file by running the following command:
+. Create the `variables.tf` file by running the following command:
```

Comment:
Add that same note:

[NOTE]
====
Copy and then edit this file *before* running the command to customize your <list whatever can be customized in this file>.
====